<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" total="15" passed="8">
  <reporter-output>
    <line>
      <![CDATA[The order number generated is :1Q11X57ME9]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="110849" started-at="2016-07-22T10:16:50Z" finished-at="2016-07-22T10:18:41Z">
    <groups>
    </groups>
    <test name="TestCase104" duration-ms="110849" started-at="2016-07-22T10:16:50Z" finished-at="2016-07-22T10:18:41Z">
      <class name="com.adactin.Testcases.TestCase116">
        <test-method status="PASS" signature="loginandSearchhotels(java.util.Hashtable)[pri:1, instance:com.adactin.Testcases.TestCase116@727eb8cb]" name="loginandSearchhotels" duration-ms="3391" started-at="2016-07-22T20:17:40Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginandSearchhotels -->
        <test-method status="FAIL" signature="CompareBookedWithItenary(java.util.Hashtable)[pri:2, instance:com.adactin.Testcases.TestCase116@727eb8cb]" name="CompareBookedWithItenary" duration-ms="11223" started-at="2016-07-22T20:17:43Z" data-provider="getPersonelData" finished-at="2016-07-22T20:17:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{BillingAddr=54 Mons road, FirstName=Johan, CVV=123.0, LastName=Cors, CreditCardNo=1.23456781234567E15, CreditCardType=VISA, ExpiryYear=2017.0, ExpiryMonth=March}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	Rooms does not match:Bookdetails=1 Room(s) Itenarydetails=1 Rooms expected [true] but found [false],
	Last name does not match:Bookdetails= Itenarydetails=Cors expected [true] but found [false],
	Room type does not match:Bookdetails=Deluxe Itenarydetails=Double expected [true] but found [false],
	Total price with gst does not match:Bookdetails=AUD $ 753.5 Itenarydetails=AUD $ 754 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	Rooms does not match:Bookdetails=1 Room(s) Itenarydetails=1 Rooms expected [true] but found [false],
	Last name does not match:Bookdetails= Itenarydetails=Cors expected [true] but found [false],
	Room type does not match:Bookdetails=Deluxe Itenarydetails=Double expected [true] but found [false],
	Total price with gst does not match:Bookdetails=AUD $ 753.5 Itenarydetails=AUD $ 754 expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.adactin.Testcases.TestCase116.CompareBookedWithItenary(TestCase116.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CompareBookedWithItenary -->
      </class> <!-- com.adactin.Testcases.TestCase116 -->
      <class name="com.adactin.Testcases.TestCase117">
        <test-method status="PASS" signature="SearchOrderId(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase117@39d9314d]" name="SearchOrderId" duration-ms="7933" started-at="2016-07-22T20:17:55Z" data-provider="Orderdata" finished-at="2016-07-22T20:18:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{OrderNumber=G353VBMMJX}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchOrderId -->
      </class> <!-- com.adactin.Testcases.TestCase117 -->
      <class name="com.adactin.Testcases.TestCase119">
        <test-method status="PASS" signature="SearchOrderId()[pri:0, instance:com.adactin.Testcases.TestCase119@b978d10]" name="SearchOrderId" duration-ms="36718" started-at="2016-07-22T20:18:02Z" finished-at="2016-07-22T20:18:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchOrderId -->
      </class> <!-- com.adactin.Testcases.TestCase119 -->
      <class name="com.adactin.Testcases.TestCase101">
        <test-method status="PASS" signature="CheckRunMode()[pri:0, instance:com.adactin.Testcases.TestCase101@2474f125]" name="CheckRunMode" is-config="true" duration-ms="12" started-at="2016-07-22T20:16:50Z" finished-at="2016-07-22T20:16:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckRunMode -->
        <test-method status="PASS" signature="LoginTest()[pri:0, instance:com.adactin.Testcases.TestCase101@2474f125]" name="LoginTest" duration-ms="2804" started-at="2016-07-22T20:16:50Z" finished-at="2016-07-22T20:16:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase101@2474f125]" name="Teardown" is-config="true" duration-ms="1336" started-at="2016-07-22T20:18:39Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="FAIL" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase102@7357a011]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <exception class="org.openqa.selenium.remote.SessionNotFoundException">
            <message>
              <![CDATA[no such session
  (Driver info: chromedriver=2.8.241075,platform=Windows NT 6.3 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 168 milliseconds
Build info: version: '2.45.0', revision: '5017cb8', time: '2015-02-27 00:00:10'
System info: host: 'DESKTOP-7I28866', ip: '192.168.2.12', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=C:\Users\sijoy\AppData\Local\Temp\scoped_dir4272_10143}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=51.0.2704.106, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 6cb2914bbcb13baca4394633e58bb836]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.SessionNotFoundException: no such session
  (Driver info: chromedriver=2.8.241075,platform=Windows NT 6.3 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 168 milliseconds
Build info: version: '2.45.0', revision: '5017cb8', time: '2015-02-27 00:00:10'
System info: host: 'DESKTOP-7I28866', ip: '192.168.2.12', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=C:\Users\sijoy\AppData\Local\Temp\scoped_dir4272_10143}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=51.0.2704.106, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 6cb2914bbcb13baca4394633e58bb836
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:162)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:622)
	at org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:463)
	at com.adactin.Testcases.TestCase102.Teardown(TestCase102.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.TestRunner.afterRun(TestRunner.java:973)
	at org.testng.TestRunner.run(TestRunner.java:604)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.SessionNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase114@3c01cfa1]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase113@40317ba2]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase119@b978d10]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase117@39d9314d]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase111@4816278d]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase116@727eb8cb]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase110@68f4865]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase115@45d2ade3]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase120@2ce6c6ec]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase103@3406472c]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase112@4eaf3684]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="SKIP" signature="Teardown()[pri:0, instance:com.adactin.Testcases.TestCase104@5717c37]" name="Teardown" is-config="true" duration-ms="0" started-at="2016-07-22T20:18:41Z" finished-at="2016-07-22T20:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
      </class> <!-- com.adactin.Testcases.TestCase101 -->
      <class name="com.adactin.Testcases.TestCase112">
        <test-method status="FAIL" signature="PriceWithGSTValidation(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase112@4eaf3684]" name="PriceWithGSTValidation" duration-ms="4216" started-at="2016-07-22T20:17:15Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Final Billed Price calculated is not proper.Actual Price:AUD $ 753.5 Expected Price:AUD $742.5 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Final Billed Price calculated is not proper.Actual Price:AUD $ 753.5 Expected Price:AUD $742.5 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.adactin.Testcases.TestCase112.PriceWithGSTValidation(TestCase112.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PriceWithGSTValidation -->
      </class> <!-- com.adactin.Testcases.TestCase112 -->
      <class name="com.adactin.Testcases.TestCase113">
        <test-method status="FAIL" signature="PastDateEntry(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase113@40317ba2]" name="PastDateEntry" duration-ms="4072" started-at="2016-07-22T20:17:20Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 Rooms is expected and actual result is 1 Room(s) expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 Rooms is expected and actual result is 1 Room(s) expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.adactin.Testcases.TestCase113.PastDateEntry(TestCase113.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PastDateEntry -->
      </class> <!-- com.adactin.Testcases.TestCase113 -->
      <class name="com.adactin.Testcases.TestCase102">
        <test-method status="PASS" signature="EnterDetailToCompareDate(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase102@7357a011]" name="EnterDetailToCompareDate" duration-ms="5922" started-at="2016-07-22T20:16:53Z" data-provider="getSearchData" finished-at="2016-07-22T20:16:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Numberroom=1-One, CheckInDate="15/05/2016", CheckOutDate="14/05/2016", Location=Sydney, Hotels=Hotel Creek, Expected_massage=Check-In Date shall be before than Check-Out Date, AdultsperRoom=4 - Four, ChildrenperRoom=2 - Two, RoomType=Double}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EnterDetailToCompareDate -->
      </class> <!-- com.adactin.Testcases.TestCase102 -->
      <class name="com.adactin.Testcases.TestCase114">
        <test-method status="PASS" signature="loginandSearchhotels(java.util.Hashtable)[pri:1, instance:com.adactin.Testcases.TestCase114@3c01cfa1]" name="loginandSearchhotels" duration-ms="3582" started-at="2016-07-22T20:17:24Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginandSearchhotels -->
        <test-method status="PASS" signature="OrderNumberVerification(java.util.Hashtable)[pri:2, instance:com.adactin.Testcases.TestCase114@3c01cfa1]" name="OrderNumberVerification" duration-ms="7752" started-at="2016-07-22T20:17:27Z" data-provider="getPersonelData" finished-at="2016-07-22T20:17:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{BillingAddr=54 Mons road, FirstName=Johan, CVV=123.0, LastName=Cors, CreditCardNo=1.23456781234567E15, CreditCardType=VISA, ExpiryYear=2017.0, ExpiryMonth=March}]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[The order number generated is :1Q11X57ME9]]>
            </line>
          </reporter-output>
        </test-method> <!-- OrderNumberVerification -->
      </class> <!-- com.adactin.Testcases.TestCase114 -->
      <class name="com.adactin.Testcases.TestCase103">
        <test-method status="FAIL" signature="PastDateEntry(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase103@3406472c]" name="PastDateEntry" duration-ms="5252" started-at="2016-07-22T20:16:59Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Numberroom=1-One, CheckInDate="11/05/2015", CheckOutDate="12/05/2015", Location=Sydney, Hotels=Hotel Creek, Expected_massage=Enter Valid dates, AdultsperRoom=4 - Four, ChildrenperRoom=2 - Two, RoomType=Double}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message not displayed even if dates are already past]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message not displayed even if dates are already past
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.adactin.Testcases.TestCase103.PastDateEntry(TestCase103.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PastDateEntry -->
      </class> <!-- com.adactin.Testcases.TestCase103 -->
      <class name="com.adactin.Testcases.TestCase104">
        <test-method status="FAIL" signature="ValidationOfData(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase104@5717c37]" name="ValidationOfData" duration-ms="3669" started-at="2016-07-22T20:17:04Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	Total Price calculated is not proper.Actual Price:AUD $ 685 Expected Price:AUD $675 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	Total Price calculated is not proper.Actual Price:AUD $ 685 Expected Price:AUD $675 expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.adactin.Testcases.TestCase104.ValidationOfData(TestCase104.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidationOfData -->
      </class> <!-- com.adactin.Testcases.TestCase104 -->
      <class name="com.adactin.Testcases.TestCase115">
        <test-method status="FAIL" signature="loginandSearchhotels(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase115@45d2ade3]" name="loginandSearchhotels" duration-ms="5009" started-at="2016-07-22T20:17:35Z" data-provider="OrderNumber" finished-at="2016-07-22T20:17:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{OrderNumber=Cancel LI5R917TTA}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 18/05/2016 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field LI5R917TTA expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 1 Rooms expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Cors expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Double expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 3 Days expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Hotel Creek expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Johan expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 15/05/2016 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field AUD $ 225 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 142997 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field AUD $ 754 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Cancel LI5R917TTA expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Sydney expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 18/05/2016 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field LI5R917TTA expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 1 Rooms expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Cors expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Double expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 3 Days expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Hotel Creek expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Johan expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 15/05/2016 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field AUD $ 225 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field 142997 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field AUD $ 754 expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Cancel LI5R917TTA expected [false] but found [true],
	The iternary details for the order: Cancel LI5R917TTA are ediatble for the field Sydney expected [false] but found [true]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.adactin.Testcases.TestCase115.loginandSearchhotels(TestCase115.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginandSearchhotels -->
      </class> <!-- com.adactin.Testcases.TestCase115 -->
      <class name="com.adactin.Testcases.TestCase110">
        <test-method status="FAIL" signature="PriceWithGSTValidation(java.util.Hashtable)[pri:0, instance:com.adactin.Testcases.TestCase110@68f4865]" name="PriceWithGSTValidation" duration-ms="3808" started-at="2016-07-22T20:17:08Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Final Billed Price calculated is not proper.Actual Price:AUD $ 685 Expected Price:AUD $675 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Final Billed Price calculated is not proper.Actual Price:AUD $ 685 Expected Price:AUD $675 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.adactin.Testcases.TestCase110.PriceWithGSTValidation(TestCase110.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.TestNG.privateMain(TestNG.java:1405)
	at org.testng.TestNG.main(TestNG.java:1374)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PriceWithGSTValidation -->
      </class> <!-- com.adactin.Testcases.TestCase110 -->
      <class name="com.adactin.Testcases.TestCase111">
        <test-method status="PASS" signature="PastDateEntry(java.util.Hashtable)[pri:1, instance:com.adactin.Testcases.TestCase111@4816278d]" name="PastDateEntry" duration-ms="3942" started-at="2016-07-22T20:17:11Z" data-provider="getSearchData" finished-at="2016-07-22T20:17:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{CheckInDate="15/05/2016", Location=Sydney, ChildrenperRoom=2 - Two, CheckOutDate="18/05/2016", RoomType=Double, AdultsperRoom=4 - Four, Numberroom=1-One, Hotels=Hotel Creek}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PastDateEntry -->
      </class> <!-- com.adactin.Testcases.TestCase111 -->
    </test> <!-- TestCase104 -->
  </suite> <!-- Suite -->
</testng-results>
